extensions/modules/file/src/main/java/org/exist/xquery/modules/file/Sync.java

```diff
@@ -88,15 +93,21 @@ public Sequence eval(final Sequence[] args, final Sequence contextSequence) thro
 		}
 		
 		final String collectionPath = args[0].getStringValue();
+
+		final String target = args[1].getStringValue();
+		Path targetDir = FileModuleHelper.getFile(target);
```

Comment:
Could this variable be `final`?

Reply:
Sure.
Btw that's only one (of the hundreds) of reasons why I prefer Kotlin to Java these days.

Reply:
Sure... But, eXist-db is in Java ;-)

Reply:
Actually - no.
targetDir is (conditionally changed) on line 117.
And it is not something I introduced -- this code has been there for a long time.
Of course, this can be avoided (as the course in "Compiler construction" teaches, every program can be converted to equivalent SSA form, but the resultant code is not necessarily more readable.

Reply:
Ah okay. No worries.